app/processing/statement/components/get-detailed-payments.js

```diff
@@ -0,0 +1,25 @@
+
+const moment = require('moment')
+const { convertToPounds } = require('../../../utility')
+
+const getDetailedPayments = async (calculation, paymentRequest, settlement) => {
+  const payments = []
+
+  const payment = {
+    invoiceNumber: calculation.invoiceNumber,
+    reference: settlement.reference,
+    dueDate: formatDate(paymentRequest.dueDate),
+    settled: formatDate(settlement.settlementDate),
+    calculated: formatDate(calculation.calculated),
+    value: convertToPounds(paymentRequest.value)
+  }
+  payments.push(payment)
+
+  return payments
+}
+
+const formatDate = (dateVal) => {

```

Comment:
Can we move this to util folder?

Reply:
I don't think so, date output format is not fixed or constant for all situation.